frontend/nav.json

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
i moved the menu data here in anticipation of creating a navbar component in react, so that we can share the data b/w react & vue.